server/controllers/reservationController.js

```diff
@@ -72,8 +71,19 @@ const testAllReservations = async (garageId) => {
   }
 }
 
-const createReservation = () => {
+const createReservation = async (reservation) => {
+  try {
+    const { data, error } = await Reservations
+      .insert(reservation)
+      .select();
+
+    if (error) throw error;
 
+    return await data;
+  } catch (error) {
+    console.log(error);
+    return null;
+  }
 }
```

Comment:
create reservation and return all reservation data?

Reply:
I'll look for another method to get my code to work